README.md

```diff
@@ -378,6 +378,104 @@ export class VehicleProductComponent extends NgxSubFormRemapComponent<OneVehicle
 
 Our "incoming" object is of type `OneVehicle` but into that component we treat it as a `OneVehicleForm` to split the vehicle (either a `speeder` or `spaceship`) in 2 **separated** properties.
 
+### Dealing with arrays
+
+When your data structure contains array(s), you may want to simply display the values in the view but chances are: You want to bind them to the form.
+
+In that case, working with a `FormArray` is the right way to go and for that, we will take advantage of the remap principles explained in the previous section.
+
+Example:
+
+```ts
+// src/app/main/listing/listing-form/vehicle-listing/people/people.component.ts#L6-L40
+
+interface PeopleForm {
```

Comment:
Update all the `people` and `person` to `crewMembers` (or just `crew`?) and `crewMember`.

Reply:
Done